src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.targets

```
@@ -267,6 +267,14 @@
     <ReadLinesFromFile File="$(_BlazorApplicationAssembliesCacheFile)" Condition="'@(_BlazorResolvedAssembly->Count())' == '0'">
       <Output TaskParameter="Lines" ItemName="_BlazorResolvedAssembly"/>
     </ReadLinesFromFile>
+
+    <ItemGroup>
+      <!--
+        Workaround for https://github.com/dotnet/aspnetcore/issues/19926. Add _BlazorResolvedAssembly to FileWrites so it doesn't
+        get removed during build incremantalism. We cannot do this in _ResolveBlazorRuntimeDependencies since it's incremental.
```

```suggestion
        get removed during build incrementalism. We cannot do this in _ResolveBlazorRuntimeDependencies since it's incremental.
```

BTW I don't follow what the comment is getting at. Could you expand it to indicate *what* removes it during incrementalism if this isn't done?